### PR TITLE
docs(installing-workflow): add followup link to register a user

### LINKED ISSUE #532: Doc issue: once installed, what's next?
Hi, I'm following this to install DEIS;
https://deis.com/docs/workflow/installing-workflow/

It's working, but doesn't says what to do next!
So I try:
https://deis.com/docs/workflow/users/registration/

But how to retrieve the controller URL?? ("supplied by your Deis administrator")

Cheers

### PR DESCRIPTION
**Pull request by bacongobbler**
This is a good follow-up step after installing Workflow.

closes #532

### CODE REVIEW

File: src/installing-workflow/index.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Installing Deis Workflow
 
 This document is aimed at those who have already provisioned a [Kubernetes v1.2 or v1.3.4+][] cluster
-and want to install Deis Workflow. If you are just getting started with Kubernetes and
-Deis Workflow, follow our [quickstart guide](../quickstart/index.md) for help.
+and want to install Deis Workflow. If help is required getting started with Kubernetes and
+Deis Workflow, follow the [quickstart guide](../quickstart/index.md) for asssistance.
```
Comment (ultimateboy) [inline src/installing-workflow/index.md]: assistance

File: src/installing-workflow/index.md [hunk 2]
```diff
@@ -92,7 +92,8 @@ Depending on the order in which the Workflow components initialize, some pods ma
 installation: if a component's dependencies are not yet available, that component will exit and Kubernetes will
 automatically restart it.
 
-Here, you can see that controller, builder and registry all took a few loops before there were able to start:
+Here, it can be seen that the controller, builder and registry all took a few loops before there were able to start:
```
Comment (ultimateboy) [inline src/installing-workflow/index.md]: I know you didnt change it, but you touched the line...

"there" should be "they"

Comment (bacongobbler) [inline src/installing-workflow/index.md]: nice catch! Just trying to remove all usage of "you" and "our", but this is a good change regardless.

File: src/installing-workflow/index.md (hunk 1 above)
Comment (bacongobbler) [inline src/installing-workflow/index.md]: changed

File: src/installing-workflow/index.md (hunk 2 above)
Comment (bacongobbler) [inline src/installing-workflow/index.md]: done

**Pull request merged**